application.py

```
@@ -0,0 +1,282 @@
+    """テンプレートフィルターの登録"""
+    @app.template_filter('datetime')
+    def format_datetime(value):
+        """日時を人間が読める形式にフォーマット"""
+        if value is None:
+            return ""
+        
+        if isinstance(value, str):
+            try:
+                from datetime import datetime
+                value = datetime.fromisoformat(value.replace('Z', '+00:00'))
+            except:
```

Ruff (0.12.2)

Do not use bare `except`

(E722)